### ISSUE TITLE
small fixes for a feasible run

### ISSUE THREAD

**Issue by msdogan** [msdogan/HOBBEStoPRMcomparison#3]
@qjhart @jrmerz Can we make following fixes, without which HEC-PRM does not run or needs debug flows? All fixes are based on comparing current *.pri file to full.pri and S09I05.pri

**PRI File updates**
- [x] Delete node 'Sink_C41'

Delete ending storage values for following nodes:
- [x] SR_ML (currently 2939.541)
- [x] SR_OL (currently 0.000)
- [x] SR_BVLB (currently 0.000)
- [x] SR_TL (currently 0.000)
- [x] SR_SS (currently 6941.0000)
- [x] GW_KRN  (currently 0.000)

Delete cost values on following links:
- [x] D884-N7 (currently 3.40282306*10^23)

**Penalty Data DSS updates**
- [x] //HXI101-EXT_REDDIN/Q(KAF)-P_EDT//FEB//
- [x] //HXI201-EXT_YUBA/Q(KAF)-P_EDT//FEB//

**Comment by qjhart** [comment]
@msdogan.  Hmm, there was one small patch that was not applied on the latest data, and you have hit on all three of those issues.   That was the node deletion, and the DSS updates.  Note, the reason, those FEB files are bad, is the CSV files
data/northwest-valley/ud_reddin/hxi101-ext_reddin/costs/FEB.csv and 
data/sacramento-river/central-basin-east/ud_yuba/hxi201-ext_yuba/costs/FEB.csv

have such small incremental changes in X, they didn't register.  The patch changes these both to 

```
capacity,cost
0,0
10000,0
```

The DSS files will then be fixed when you run the build command

**Comment by qjhart** [comment]
Regarding the ending storage values, are you saying these are wrong ;SR_ML,SR_OL,SR_BVLB,SR_TL?  These are the values that presumably came from the S09I05 database.  Is it possible if the value is zero, we are supposed to not include it in the pri file? 

The D884-N7 link is unusual.  In the access database this is a very big number 3.40E38.

**Comment by qjhart** [comment]
@msdogan, I tried my small test, that I have run before, using these fixes, and I still require a few DBUG nodes.   The example below runs, eg.

```
# pull and update your calvin-node-tools.
prm --debug=SR_GNT,SR_HID,SR_ML --prefix=wy2002 --start=2002-10 --stop=2003-09 build
perl -i -p -e 's/OCT1921/OCT2002/' wy2002.pri
prm --prefix=wy2002 run
```

**Comment by msdogan** [comment]
@qjhart SR_ML,SR_OL,SR_BVLB,SR_TL are natural lakes and we don't specify end-of-storage requirement for them. These are also sinks in the system, and only outlet is the evaporation and some of them we don't even calculate evaporation. So, if we specify end-of-period requirements, it results in debug flows (source or probably sink). There is one more node that I forgot to add and fix: SR_SS. This also should not have EOP storage. We don't have EOP storage values in S09I05.pri, but somehow EOP storage is assigned to initial storage (might this be a bug while reading data?). And yes, even with that fixes, I got 0.22 TAF debug flow in Mono Basin. I was doing statewide runs for whole period `prm build --prefix test_dbg --debug All` and `prm run --prefix test_dbg`
I am having debug flows for DBUGSRC-C35, DBUGSRC-N201 and DBUGSRC-SR_GNT. For C35 and N201, there was some calibration flows that I added. Those can be seen in S09I05.pri. They are constrained flows, and they are not added when buildding test_dbg.pri. @jrmerz can we add those constrained flows as well? Links for them should be CN2-C35 and CN2-N201. Only location we need to resolve is SR_GNT. We have a minimum in-stream flow requirements and it is causing a problem for some month. But total debug flow is very small, 0.22 TAF.
[S09I05.zip](https://github.com/msdogan/HOBBEStoPRMcomparison/files/350470/S09I05.zip)
